nautobot/extras/tests/test_filters.py

```diff
@@ -1669,6 +1368,12 @@ def test_destination_type(self):
 class RelationshipAssociationFilterSetTestCase(FilterTestCases.FilterTestCase):
     queryset = RelationshipAssociation.objects.all()
     filterset = RelationshipAssociationFilterSet
+    generic_filter_tests = (
+        ("destination_id",),
+        # ("relationship", "relationship__id"),  # TODO?
+        ("relationship", "relationship__key"),
```

Comment:
Is there a reason preventing us from changing `relationship` from `ModelMultipleChoiceFilter` to`NaturalKeyOrPKMultipleChoiceFilter`?

Reply:
I don't think so, no, just wanted to keep the scope of this PR relatively constrained.